testsuite/features/secondary/min_change_software_channel.feature

```diff
@@ -5,6 +5,7 @@
 # - features/secondary/allcli_software_channels.feature
 # If "SLE15-SP4-Installer-Updates for x86_64" fails to be unchecked
 
+# This test fails on github validation
```

Comment:
With the changes made below, this feature still failing on GH validation?
If still failing, I'd better revert the changes, to don't add any additional risk to new reds.

Reply:
Ok. So then, my only suggestion is to please add the reason why we use different channels on this and other features, as part of the PR description.

Reply:
I edited the description.